python-threatexchange/threatexchange/exchanges/impl/techagainstterrorism_api.py

```diff
@@ -10,17 +10,26 @@
 from threatexchange.exchanges import fetch_state as state
 from threatexchange.exchanges import signal_exchange_api
 from threatexchange.exchanges import auth
-from threatexchange.exchanges.collab_config import (
-    CollaborationConfigWithDefaults,
-)
+from threatexchange.exchanges.collab_config import CollaborationConfigWithDefaults
 from threatexchange.signal_type.signal_base import SignalType
 from threatexchange.signal_type.pdq.signal import PdqSignal
 from threatexchange.signal_type.md5 import VideoMD5Signal
 
+
+@dataclass
+class TATCheckpoint(state.FetchCheckpointBase):
```

Comment:
Consider also implementing the method for showing the "after" time, which will help your debugging: 

https://github.com/facebook/ThreatExchange/blob/main/python-threatexchange/threatexchange/exchanges/fetch_state.py#L41

Reply:
Hi David,

Our checkpoint isn't a timestamp, its a string with a timestamp and content id joined by a comma eg, `"123456,123"` when passing said value to `get_progress_timestamp` it throws an error due to the checkpoint not being an integer.